frontend/src/components/new-sightings-page/NewSightingForm.tsx

```diff
@@ -67,6 +67,23 @@ export const NewSightingForm: React.FC<NewSightingFormProps> = ({
 
   const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setLocationInputType(event.target.value as LocationInputType);
+    setFormValues({
+      ...formValues,
+      longitude: "",
+      latitude: "",
+    });
+  };
+
+  const getLocationFromPhone = () => {
```

Comment:
Not necessarily a phone:
```suggestion
  const getLocationFromBrowser = () => {
```